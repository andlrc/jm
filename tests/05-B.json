{
	"@extends": ["05-A.json"],
	"array": [
		{
			"@append": true,
			"my_func": my_global_factory({
				"a": 1,
				"b": 2,
				"c": [
					{"prop": 1},
					{"prop": 2},
					{"prop": 3}
				]
			})
		},
		{
			"hello": function() {

			},
			"a": 1
		},
		(function() {
			return -1;
		})()
	]
}
