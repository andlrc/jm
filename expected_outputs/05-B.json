{
	"array": [
		{
			"fn": function(a, b) {
				return {
					"a": a,
					"b": b
				};
			}(1, 2)
		,
			"hello": function() {

			},
			"a": 1
		},
		(function() {
			return -1;
		})()
	,
		{
			"my_func": my_global_factory({
				"a": 1,
				"b": 2,
				"c": [
					{"prop": 1},
					{"prop": 2},
					{"prop": 3}
				]
			})
		
		}
	]
}
